src/imu/imu.h

```diff
@@ -0,0 +1,62 @@
+#ifndef _IMU_H_
+#define _IMU_H_
+
+#include <yarp/dev/PolyDriver.h>
+#include <yarp/dev/MultipleAnalogSensorsInterfaces.h>
+#include <yarp/dev/IPositionControl.h>
+#include <yarp/dev/IEncoders.h>
+#include <yarp/dev/IAxisInfo.h>
+#include <yarp/dev/IControlLimits.h>
+#include <yarp/robottestingframework/TestCase.h>
+
+#include <iDynTree/KinDynComputations.h>
+#include <iDynTree/ModelLoader.h>
+#include <iDynTree/Model.h>
+
+class Imu : public yarp::robottestingframework::TestCase {
+    public:
+        Imu();
+        virtual ~Imu();
+        virtual bool setup(yarp::os::Property& property);
+        virtual void tearDown();
+        virtual void run();
+
+    private:
+        std::string robotName;
+        std::string portName;
+        std::string modelPath;
+        std::string frameName;
+        std::string sensorName;
+
+        yarp::dev::PolyDriver *MASclientDriver;
+        yarp::dev::PolyDriver *controlBoardDriver;
```

Comment:
I don't think there is the need to use pointers here, you can use a plain `Polydriver`, this will remove the `new` and `delete` from the code